### PR TITLE
Self-Sovereign Identity

### PR DESCRIPTION
**Pull request by InvictusRMC**
# Introduction
This PR adds the 18 plus app and updates the Kotlin version used.

### CODE REVIEW

**Comment by synctext** [comment]
Checks are failing. Can you update this PR? 
We're running towards a new release for EU-wide publicity before 8 April. @InvictusRMC @xoriole

**Comment by InvictusRMC** [comment]
Please merge [kotlin-ipv8](https://github.com/Tribler/kotlin-ipv8/pull/43) first as this build will pass if that one is merged (and the submodule is updated).

**Comment by synctext** [comment]
New student team at bsc level is starting after next week. So we need to get this polished and passing. @xoriole  :+1:

**Comment by synctext** [comment]
@InvictusRMC @xoriole this PR will be repaired coming days.

**Comment by InvictusRMC** [comment]
That was annoying to fix, some dependency hell: MusicDAO was using an old version of coroutines, clashing with the newer ones used. Their tests only work on Linux. so I had to set up a VM for this project. But everything seems to function as expected.

@xoriole it should be good to go.

File: ig-ssi/src/androidTest/java/nl/tudelft/trustchain/explorer/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package nl.tudelft.trustchain.explorer
```
Comment (xoriole) [inline ig-ssi/src/androidTest/java/nl/tudelft/trustchain/explorer/ExampleInstrumentedTest.kt:1]: This example test file can be removed

File: ig-ssi/src/test/java/nl/tudelft/trustchain/explorer/ExampleUnitTest.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.trustchain.explorer
```
Comment (xoriole) [inline ig-ssi/src/test/java/nl/tudelft/trustchain/explorer/ExampleUnitTest.kt:1]: This test does not add any value, please remove it

**Review by xoriole — CHANGES_REQUESTED** [review]
Please remove the unnecessary example test files

**Comment by InvictusRMC** [comment]
@xoriole thank you for the review. I've implemented your requested changes.

**Review by xoriole — APPROVED** [review]

**Pull request merged**